chart-pie.js

```diff
@@ -93,6 +97,37 @@ class ChartPie extends ResizeBehavior(ContextBehavior(ChartPropertyBehavior(mixi
       }]
     };
   }
+
+  // Source: https://stackoverflow.com/questions/33363373/how-to-display-pie-chart-data-values-of-each-slice-in-chart-js
+  _drawValuesInPie() {
+    const {ctx} = this.chart;
+    ctx.font = Chart.helpers.fontString(Chart.defaults.global.defaultFontFamily, 'normal', Chart.defaults.global.defaultFontFamily);
+    ctx.textAlign = 'center';
+    ctx.textBaseline = 'bottom';
+    ctx.fillStyle = '#fff';
+
+    this.data.datasets.forEach(function (dataset) {
+
+      for (let i = 0; i < dataset.data.length; i++) {
```

Comment:
any practical reason why we use a `forEach` above and a `for` loop inside of it?

Reply:
in the inside loop we're using the iteration value in different cases, we can change it to forEach but it is a little bit tricky